internal/statesync/stateprovider.go

```diff
@@ -402,6 +394,11 @@ func (s *stateProviderP2P) consensusParams(ctx context.Context, height int64) (p
 							Height: uint64(height),
 						},
 					}); err != nil {
+						// this only errors if
```

Comment:
I have eaten
the errors
that were in
the context

and which
you were probably
planning
to propagate

Forgive me
they were irrelevant
so late
and so contextual